contrib/geoToolbox.lua

```diff
@@ -0,0 +1,590 @@
+--[[
+    This file is part of darktable,
+    copyright (c) 2016 Tobias Jakobs
+
+    darktable is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    darktable is distributed in the hope that it will be useful,
+     but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with darktable.  If not, see <http://www.gnu.org/licenses/>.
+]]
+--[[
+
+USAGE
+* require this script from your main lua file
+  To do this add this line to the file .config/darktable/luarc: 
+require "geoToolbox"
+
+* it creates a new geoToolbox lighttable module
+]]
+
+local dt = require "darktable"
+local df = require "lib/dtutils.file"
+local gettext = dt.gettext
+
+dt.configuration.check_version(...,{3,0,0},{4,0,0})
+
+-- Tell gettext where to find the .mo file translating messages for a particular domain
+gettext.bindtextdomain("geoToolbox",dt.configuration.config_dir.."/lua/")
+
+local function _(msgid)
+    return gettext.dgettext("geoToolbox", msgid)
+end
+
+-- <GUI>
+local labelDistance = dt.new_widget("label")
+labelDistance.label = "Distance:"
+
+local labelCopyGPSlat = dt.new_widget("check_button")
+{
+  label = "latitude:", 
+  value = true
+}
+local labelCopyGPSlon = dt.new_widget("check_button")
+{
+  label = "longitude:", 
+  value = true
+}
+local labelCopyGPSele = dt.new_widget("check_button")
+{
+  label = "elevation:", 
+  value = true
+}
+-- </GUI> 
+
+local function selectWithGPS()
```

Comment:
I just tried this on my system.  There were 11136 images with GPS information.  The lighttable view keeps going back and forth between graying out and color.  It acts like it's choking to death on the number of images.  :-)  Would it make more sense for this to have parameters to limit the scope of the search and return a more manageable number of images?  Or maybe use collect images to gather a collection and then just work over that?